validator/accounts/v2/wallet/wallet.go

```diff
@@ -130,7 +130,8 @@ func IsValid(walletDir string) (bool, error) {
 	}
 	f, err := os.Open(expanded)
 	if err != nil {
-		if strings.Contains(err.Error(), "no such file") {
+		if strings.Contains(err.Error(), "no such file") ||
+			strings.Contains(err.Error(), "cannot find the path") {
 			return false, nil
 		}
```

Comment:
If os.Open returns an error, that error is always *os.PathError so it represents more error states than just "file not found."  os.PathError is a struct which includes member Err of interface type error.  In both Linux and Windows, the concrete type of Err would be syscall.Errno (a number).  The actual value in Windows is ERROR_PATH_NOT_FOUND and this is unfortunately Windows specific defined in syscall/types_windows.go.  So I don't know if there is a clean way to capture "file not found" type of error all in one check.  So I just test for the string output of the error.